src/components/Transaction/index.tsx

```diff
@@ -2,17 +2,35 @@ import React from 'react'
 import { Link } from 'react-router-dom'
 import InputOutputIcon from '../../assets/input_arrow_output.png'
 import { parseDate } from '../../utils/date'
-import { shannonToCkb } from '../../utils/util'
-import CellContainer from './CellList'
+import { shannonToCkb, formattorConfirmation } from '../../utils/util'
+import PaginationList from './PaginationList'
 import {
   ConfirmationCapacityContainer,
   HashBlockContainer,
   InputOutputContainer,
   MainContainer,
   Separate,
 } from './styled'
-import { CELL_PAGE_SIZE } from './utils/const'
-import { formattorConfirmation, getCapacityChange } from './utils/utils'
+import TransactionCell from './Cell'
+import { parseNumber } from '../../utils/number'
+
+export const CELL_PAGE_SIZE = 10
+
+const getCapacityChange = (transaction: { display_inputs: [any]; display_outputs: [any] }, address?: string) => {
```

Comment:
`transaction` is no need and please add data type for `display_inputs ` and `display_outputs `